src/fifo.ts

```diff
@@ -57,41 +57,32 @@ export interface FIFOOptions {
    * When the queue reaches this size, it will be split into head/tail parts
    */
   splitLimit?: number
-
-  /**
-   * If true, `size` will be the number of items in the queue. If false, all
-   * values will be interpreted as Uint8Arrays and `size` will be the total
-   * number of bytes in the queue.
-   */
-  objectMode?: boolean
 }
 
 export class FIFO<T> {
   public size: number
   private readonly hwm: number
   private head: FixedFIFO<T>
   private tail: FixedFIFO<T>
-  private readonly objectMode: boolean
 
   constructor (options: FIFOOptions = {}) {
     this.hwm = options.splitLimit ?? 16
     this.head = new FixedFIFO<T>(this.hwm)
     this.tail = this.head
     this.size = 0
-    this.objectMode = Boolean(options.objectMode)
+  }
+
+  calculateSize (obj: any): number {
+    if (obj.byteLength != null) {
```

Comment:
```suggestion
    if (obj && typeof obj == "object" && 'byteLength' in obj && obj.byteLength != null) {
```

Reply:
If `obj` is null/undefined, what does that mean in terms of it's size?